gwpy/table/io/gravityspy.py

```diff
@@ -92,10 +92,23 @@ def get_gravityspy_triggers(tablename, engine=None, **kwargs):
 
 def get_connection_str(db='gravityspy',
                        host='gravityspy.ciera.northwestern.edu',
-                       user=os.getenv('GRAVITYSPY_DATABASE_USER', None),
-                       passwd=os.getenv('GRAVITYSPY_DATABASE_PASSWD', None)):
+                       user=None,
+                       passwd=None):
     """Create string to pass to create_engine
+
+    Note:
+        If not supplied, user and passwd are
+        resolved by scraping your environement for
+        ``GRAVITYSPY_DATABASE_USER`` and
+        ``GRAVITYSPY_DATABASE_PASSWD``.
+
+        *Both* user and passwd are required otherwise
+        it is ignored.
```

Comment:
@scottcoughlin2014, can you add `Parameters` and `Returns` sections in the docstring, see other functions for examples?

Also, the final sentence here, I think, is ambiguous. Perhaps something like (adding sphinx formatting):

```
    .. note::

       `user` and `passwd` should be given together, otherwise they will be
       ignored and values will be resolved from the ``GRAVITYSPY_DATABASE_USER``
       and ``GRAVITYSPY_DATABASE_PASSWD`` environment variables.